<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Aluno</class>
 <widget class="QWidget" name="Aluno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableAluno">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>idAluno</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nomeAluno</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>matricula</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Manipulação</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>304</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelNomeAluno">
         <property name="text">
          <string>Nome do Professor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linenomeAluno"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelMatricula">
         <property name="text">
          <string>Matrícula</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineMatricula"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>151</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnInserir">
       <property name="text">
        <string>Inserir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelBusca">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineBusca"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
